src/main.rs

```diff
@@ -174,33 +186,32 @@ fn decode_backup<R: Read>(mut reader: R, password: &[u8], attachment_folder: &Pa
 			read_frame(&mut reader, &mut cipher_data, verify_mac)?;
 		seek_position += consumed_bytes;
 		let frame = protobuf::parse_from_bytes::<Backups::BackupFrame>(&frame_content)
-			.expect(&format!("Could not parse frame from {:?}", frame_content));
+			.unwrap_or_else(|_| panic!("Could not parse frame from {:?}", frame_content));
```

Comment:
Why did you switch to unwrap_or_else here?

Reply:
Many thanks for your remarks! Actually, I don't have a strong justification for this change. Mainly, I did it due to clippy warnings. This is explained here: https://rust-lang.github.io/rust-clippy/master/index.html#expect_fun_call
However, it seems that the format function will be called with expect even if the result ist `Ok`.

Reply:
While putting the different functions into modules, I decided to use `anyhow`. So this will be changed anyway in my following pull request, if you agree to the use of anyhow.
Link to anyhow: https://docs.rs/anyhow/1.0.31/anyhow/. Anyhow is the successor of the `failure` crate.

Reply:
Ah okay, that makes sense. Switching to anyhow seems good to me, since error-chain is deprecated by now IIRC